R/lib.R

```diff
@@ -174,9 +173,6 @@ get_druggability_data <- function(id) {
 #' @export
 process_druggability_data <- function(data, gene_info) {
   data %>%
-    assertr::chain_start() %>%
-    assertr::verify(ensembl_gene_id %in% gene_info$ensembl_gene_id) %>%
```

Comment:
This assertion fails because there isn't a gene id present in gene_info for all the ones in the druggability data. @kdaily, are there any other checks that you think make since here?